test/src/main/java/org/apache/accumulo/test/fate/MultipleStoresTestRunner.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.test.fate;
+
+import java.util.function.Predicate;
+
+import org.apache.accumulo.core.fate.FateStore;
+import org.apache.accumulo.core.fate.zookeeper.ZooUtil;
+import org.apache.zookeeper.KeeperException;
+
+public interface MultipleStoresTestRunner {
```

Comment:
Could we remove this interface and move the methods it declares into MultipleStoresIT as abstract methods?  No need to do that, just want to make sure I understand the structure of these changes.  Wondering if this interface has any other purposes.

Reply:
yeah, not used anywhere else. Moved into MultipleStoresIT